coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -51,15 +63,15 @@ interface ImageLoader {
      * @param request The request to execute.
      * @return A [RequestDisposable] which can be used to cancel or check the status of the request.
      */
-    fun load(request: LoadRequest): RequestDisposable
+    fun launch(request: LoadRequest): RequestDisposable
 
     /**
      * Load the [request]'s data and suspend until the operation is complete. Return the loaded [Drawable].
      *
      * @param request The request to execute.
      * @return The [Drawable] result.
      */
-    suspend fun get(request: GetRequest): Drawable
+    suspend fun launch(request: GetRequest): Drawable
```

Comment:
Is it idiomatic to overload coroutines names like launch? Is there a situation where this could be ambiguous?

Otherwise I like this name,  though execute() would also work! With coroutines `execute()` is the suspending form and you never need to choose between blocking and non-blocking!

Reply:
Another word for this is `load()` since that’s what an ImageLoader does.

Reply:
@swankjesse Great point. Overloading `launch` could be confusing. There might be cases where you'd do something like:

```kotlin
CoroutineScope(Dispatchers.Main.immediate).launch {
    val request = GetRequest.Builder().data(url).build()
    val drawable = imageLoader.launch(request)
}
```

`execute` doesn't collide with anything (that I'm aware of) and is usable for both `GetRequest`s and `LoadRequest`s.

`load` works well for `LoadRequest`s, but I think it's awkward to also `load` a `GetRequest`. I'm currently using `ImageLoader.load` and `ImageLoader.get` to execute requests but I'm starting to think having one name to execute both is preferrable...